addons/resource.language.en_gb/resources/strings.po

```diff
@@ -13808,6 +13808,11 @@ msgctxt "#20419"
 msgid "Replace file names with library titles"
 msgstr ""
 
+#: system/settings/settings.xml
+msgctxt "#29995"
+msgid "Parse movies in 'Disc n' subdirectories"
```

Comment:
Now that we support Disc, Disk and localized version according to Kodi language settings, you need to extend this I guess.

Reply:
I've extended the description that appears at the bottom.